src/storage-control/tests/mocking.rs

```diff
@@ -249,53 +249,60 @@ mod test {
         let _ = client.get_operation().send().await.unwrap_err();
     }
 
-    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
-    async fn default_stub() {
-        #[derive(Debug)]
-        struct DefaultStorageControl;
-        impl gcs::stub::StorageControl for DefaultStorageControl {}
-
-        let client = gcs::client::StorageControl::from_stub(DefaultStorageControl);
-
-        let _ = client.delete_bucket().send().await.unwrap_err();
-        let _ = client.get_bucket().send().await.unwrap_err();
-        let _ = client.create_bucket().send().await.unwrap_err();
-        let _ = client.list_buckets().send().await.unwrap_err();
-        let _ = client
-            .lock_bucket_retention_policy()
-            .send()
-            .await
-            .unwrap_err();
-        let _ = client.get_iam_policy().send().await.unwrap_err();
-        let _ = client.set_iam_policy().send().await.unwrap_err();
-        let _ = client.test_iam_permissions().send().await.unwrap_err();
-        let _ = client.update_bucket().send().await.unwrap_err();
-        let _ = client.compose_object().send().await.unwrap_err();
-        let _ = client.delete_object().send().await.unwrap_err();
-        let _ = client.restore_object().send().await.unwrap_err();
-        let _ = client.get_object().send().await.unwrap_err();
-        let _ = client.update_object().send().await.unwrap_err();
-        let _ = client.list_objects().send().await.unwrap_err();
-        let _ = client.rewrite_object().send().await.unwrap_err();
-        let _ = client.move_object().send().await.unwrap_err();
+    macro_rules! default_stub_method {
+        ($($method:ident),*) => {
+            $(
+                #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+                #[should_panic]
+                async fn $method() {
+                    #[derive(Debug)]
+                    struct DefaultStorageControl;
+                    impl gcs::stub::StorageControl for DefaultStorageControl {}
```

Comment:
optional nit: move outside of the macro?

Reply:
Done.